app/components/provider_interface/application_card_component.rb

```diff
@@ -18,7 +18,21 @@ def initialize(application_choice:)
 
     def days_to_respond_text
       if (days_left_to_respond = application_choice.days_left_to_respond)
-        "#{days_until(Date.current + days_left_to_respond).capitalize} to respond"
+        if days_left_to_respond.zero?
+          'Last day to make decision'
+        else
+          "#{days_until(Date.current + days_left_to_respond).capitalize} to make decision"
+        end
+      end
+    end
+
+    def decision_by_date_text
```

Comment:
Maybe provide a bit more context with the method name, e.g. `candidate_days_to_respond_text`?

Reply:
👍